forge-gui/res/cardsfolder/upcoming/redemptor_dreadnought.txt

```diff
@@ -0,0 +1,12 @@
+Name:Redemptor Dreadnought
+ManaCost:5
+Types:Artifact Creature Astartes Dreadnought
+PT:4/4
+K:Trample
+A:SP$ PermanentCreature | Cost$ 5 ExileFromGrave<X/Creature/creature card> | XMaxLimit$ 1
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Plasma Incinerator — Whenever CARDNAME attacks, if a card is exiled with it, it gets +X/+X until end of turn, where X is the power of the exiled card.
```

Comment:
Because of the intervening if, this should have some kind of check or condition, right? Otherwise it will always trigger